src/phantom/predicates/interval.py

```diff
@@ -17,49 +17,49 @@
 T = TypeVar("T")
 
 
-def open(low: T, high: T) -> Predicate[SupportsLeGe[T]]:
+def open(low: T, high: T) -> Predicate[SupportsLtGt[T]]:
```

Comment:
I believe we should rename `open` and `closed` in this module too, as they are part of the public API of the library.